docs/reference/components/sources/logplex.cue

```diff
@@ -56,6 +56,33 @@ components: sources: logplex: {
 				examples: ["0.0.0.0:80"]
 			}
 		}
+		auth: {
```

Comment:
If this file largely inherits from the `http` source you can simply reference the `http` cue data. For example:

https://github.com/timberio/vector/blob/d6b4ed40e993059839c2d0f42c3ada69c4590325/docs/reference/components/sources/syslog.cue#L8-L12